src/csvcubed/utils/sparql_handler/sparql_queries/select_codelist_csv_url.sparql

```diff
@@ -3,8 +3,8 @@ PREFIX csvw: <http://www.w3.org/ns/csvw#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 
 
-SELECT DISTINCT ?conceptScheme ?tableUrl
-WHERE {   
+SELECT DISTINCT ?conceptScheme ?tableUrl ?conceptSchemeUrlStr
```

Comment:
Pretty sure I added this `?conceptSchemeUrlStr` last week while playing around with the query. We should make sure we definitely dont need it